h5pt_test.go

```diff
@@ -5,109 +5,261 @@
 package hdf5
 
 import (
+	"fmt"
 	"os"
 	"reflect"
 )
 import "testing"
 
 const (
-	fname    string = "ex_table_01.h5"
-	tname    string = "table"
-	nrecords int    = 8
+	fname     string = "ex_table_01.h5"
+	tname     string = "table"
+	chunkSize int    = 10
+	compress  int    = 0
 )
 
 type particle struct {
-	// name        string  `hdf5:"Name"`      // FIXME(sbinet)
+	//name        string  `hdf5:"Name"`		 // FIXME(TacoVox)
+	vehicle_no  uint8   `hdf5:"Vehicle Number"`
+	sattelites  int8    `hdf5:"Sattelites"`
+	cars_no     int16   `hdf5:"Number of Cars"`
+	trucks_no   int16   `hdf5:"Number of Trucks"`
+	min_speed   uint32  `hdf5:"Minimum Speed"`
 	lati        int32   `hdf5:"Latitude"`
+	max_speed   uint64  `hdf5:"Maximum Speed"`
 	longi       int64   `hdf5:"Longitude"`
 	pressure    float32 `hdf5:"Pressure"`
 	temperature float64 `hdf5:"Temperature"`
+	accurate    bool    `hdf5:"Accurate"`
 	// isthep      []int                     // FIXME(sbinet)
 	// jmohep [2][2]int64                    // FIXME(sbinet)
 }
 
-func TestTable(t *testing.T) {
+func testTable(dType interface{}, data interface{}, nrecords int) (err error) {
```

Comment:
`testTable` should probably take a `t *testing.T` parameter instead of return an error.

this would make for a better debugging experience: calling `t.Errorf` or `t.Fatalf` instead of return an error and thus getting a full stack trace.

Reply:
Hmm yeah. Maybe you are right. Will adapt it accordingly.